pom.xml

```diff
@@ -208,7 +208,7 @@
     <vers.hdp.solr>5.2.1</vers.hdp.solr>
     <solr.hdp.expected.version>5.2.1</solr.hdp.expected.version> <!-- sanity check to verify we actually run against the expected version rather than some outdated version -->
     <vers.hdp.tika>1.7</vers.hdp.tika>
-    <vers.hdp.jackson>2.3.1</vers.hdp.jackson>
+    <vers.hdp.jackson>2.7.4</vers.hdp.jackson>
```

Comment:
Good question. So, there are two implementations/versions of jackson and for some reason, we use both. This version is the faster XML version so this update is to harmonize with that version in Rocana Ops:
```xml
    <!-- Faster XML version -->
    <version.fasterxml-jackson>2.7.4</version.fasterxml-jackson>
    <!-- ... -->
    <!-- Other jackson -->
    <version.jackson>1.9.13</version.jackson>
```